feat: rewrite the Creating your bot and Slash command sections (#1281)
discordjs/guide: guide/slash-commands/parsing-options.md

# Parsing options

## Command options

In this section, we'll cover how to access the values of a command's options. Let's consider a `ban` command example with two options:

```js {6-14}
const { SlashCommandBuilder, PermissionFlagsBits } = require('discord.js');

module.exports = {
	data: new SlashCommandBuilder()
		.setName('ban')
		.setDescription('Select a member and ban them.')
		.addUserOption(option =>
			option
				.setName('target')
				.setDescription('The member to ban')
				.setRequired(true))
		.addStringOption(option =>
			option
				.setName('reason')
				.setDescription('The reason for banning'))
		.setDefaultMemberPermissions(PermissionFlagsBits.BanMembers)
		.setDMPermission(false),
};
```

> In the execute method, we'll get the value of these two options from the `CommandInteractionOptionResolver` as shown below:

Reviewer: ```suggestion
In the execute method, you can retrieve the value of these two options from the `CommandInteractionOptionResolver` as shown below:
```
